nbopen.py

```diff
@@ -47,7 +49,11 @@ def nbopen(filename, profile='default'):
             nbdir = os.path.dirname(filename)
 
         print("Starting new server")
-        notebookapp.launch_new_instance(file_to_run=os.path.abspath(filename),
+        path = os.path.relpath(filename, start=nbdir)
+        notebook_url = url_path_join('notebooks', url_escape(path))
+        if query:
+            notebook_url = notebook_url + query
+        notebookapp.launch_new_instance(default_url=notebook_url,
```

Comment:
I understand, this is behaviour which affects a number of aspects.

I will investigate a plausible alternative